Document Provider Service Request/Response Models (#600)
trinsic-id/sdk: docs/reference/services/provider-service.md

### Invite Participant

> Users can be added as participants in the ecosystem by sending an invitation and a security code. This code can be sent directly through them using existing platforms or via email, SMS, etc.

Reviewer: ```suggestion
Users can be added as participants in an ecosystem by sending an invitation and a security code. This code can be sent directly to the invitee using existing platforms or via email, SMS, etc.
```
